<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Application_Window</class>
 <widget class="QWidget" name="Main_Application_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>675</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>675</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>675</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Ericsson Hilda</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>CLI Automation</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Application_GUI.qrc">
    <normaloff>:/Main_Application_window/ericsson-blue-icon-logo.ico</normaloff>:/Main_Application_window/ericsson-blue-icon-logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/Main_Application_window/wp3610575-automation-wallpapers-1080x675-copy.png);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background:transparent;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="font">
         <font>
          <family>Ericsson Hilda</family>
          <pointsize>25</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="title">
         <string>MPBN Nokia CLI Automation</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="4" column="0">
          <layout class="QVBoxLayout" name="table_verticalLayout">
           <item>
            <widget class="QTableView" name="task_database_tableview">
             <property name="font">
              <font>
               <family>Ericsson Hilda</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">#task_database_tableview{
color:black;
	background-color: rgb(190, 190, 190);
	alternate-background-color: rgb(210, 210, 210);
}
</string>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="dragDropMode">
              <enum>QAbstractItemView::InternalMove</enum>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
             <property name="gridStyle">
              <enum>Qt::NoPen</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="selected_host_details_horizontal_layout">
           <item>
            <widget class="QLabel" name="selected_host_details_label">
             <property name="font">
              <font>
               <family>Ericsson Hilda</family>
               <pointsize>16</pointsize>
               <bold>true</bold>
               <underline>true</underline>
              </font>
             </property>
             <property name="text">
              <string>Selected Host Details :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="selected_host_details_line_edit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Ericsson Hilda</family>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="whatsThis">
              <string extracomment="Path of the Selected Host details"/>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">#selected_host_details_line_edit{
background-color: rgb(220, 220, 220);
color: rgb(124, 124, 124);
border-radius: 8px;
padding:2px;}
</string>
             </property>
             <property name="text">
              <string> Host Details: -</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="5" column="0">
          <widget class="QFrame" name="frame_for_session_buttons">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="sizeConstraint">
               <enum>QLayout::SetFixedSize</enum>
              </property>
              <item>
               <widget class="QPushButton" name="new_session_button">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>100</horstretch>
                  <verstretch>100</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>1000</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>1000</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="styleSheet">
                 <string notr="true">#new_session_button{
color:white;
background:solid;
background-color: rgb(22, 15, 121);
padding:2px;
border:1px solid;
border-color: white;
border-radius:12px;
	font: 14pt &quot;Ericsson Hilda&quot;;
}

#new_session_button:hover{
background-color:rgb(255,255,255);
border:1px solid;
border-color:rgb(22, 15,121 );
color:rgb(22, 15, 121);
	font: 700 14pt &quot;Ericsson Hilda&quot;;
}

#new_session_button:pressed{
background-color:rgb(220, 220, 220);
border:1px solid;
border-color:rgb(15, 13, 125);
color:rgb(22, 15, 121);
	font: 800 14pt &quot;Ericsson Hilda&quot;;
}</string>
                </property>
                <property name="text">
                 <string>New Session</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QFrame" name="frame_for_vendor">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="vendor_selection_details_horizontal_layout">
              <property name="bottomMargin">
               <number>10</number>
              </property>
              <item>
               <widget class="QComboBox" name="vendor_details_selection_combobox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>220</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>220</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="baseSize">
                 <size>
                  <width>200</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Ericsson Hilda</family>
                  <pointsize>14</pointsize>
                  <stylestrategy>PreferDefault</stylestrategy>
                  <hintingpreference>PreferFullHinting</hintingpreference>
                 </font>
                </property>
                <property name="focusPolicy">
                 <enum>Qt::WheelFocus</enum>
                </property>
                <property name="contextMenuPolicy">
                 <enum>Qt::ActionsContextMenu</enum>
                </property>
                <property name="acceptDrops">
                 <bool>false</bool>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">#vendor_details_selection_combobox{
color:black;
border-radius:5px;
border:1px solid;
background:transparent;
	background-color: rgb(215, 215, 215);
	alternate-background-color: rgb(189, 189, 189);
}

#vendor_details_selection_combobox::drop-down:button{
border-top-radius:5px;
border-bottom-radius:5px;
	border-image: url(:/Main_Application_window/drop-downarrow.png);
}

#vendor_details_selection_combobox QAbstractItemView{
outline:none;
background-style:solid;
}

#vendor_details_selection_combobox QListView{
color:black;
}

#vendor_details_selection_combobox QListView:item{
color:black;
background-color: rgb(190, 190, 190);
}

#vendor_details_selection_combobox QListView:item:selected{
color:black;
background-style:solid;
background-color: rgb(220, 220, 220);
outline:none;}

</string>
                </property>
                <property name="currentIndex">
                 <number>-1</number>
                </property>
                <property name="placeholderText">
                 <string>Select the vendor</string>
                </property>
                <property name="frame">
                 <bool>true</bool>
                </property>
                <property name="modelColumn">
                 <number>0</number>
                </property>
                <item>
                 <property name="text">
                  <string>New Item</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>New Item</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QFrame" name="sheet_Creater_status_frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1">
             <layout class="QVBoxLayout" name="sheet_creater_status_vertical_layout">
              <item>
               <widget class="QLabel" name="sheet_creater_status_button_label">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>500</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>500</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">#sheet_creater_status_button_label{
color:white;
background:solid;
background-color: rgb(22, 15, 121);
padding:2px;
border:1px solid;
border-color: white;
border-radius:12px;
	font: 14pt &quot;Ericsson Hilda&quot;;
}
</string>
                </property>
                <property name="text">
                 <string>Sheet Creater Task Status</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="sheet_creater_task_status_label">
                <property name="font">
                 <font>
                  <family>Ericsson Hilda</family>
                  <pointsize>14</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="2">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QFrame" name="frame_for_buttons_and_their_status">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="buttons_and_status_vertical_status">
              <item>
               <layout class="QHBoxLayout" name="buttons_horizontal_layout">
                <item>
                 <widget class="QPushButton" name="template_checks_btn">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">#template_checks_btn{
color:white;
background:solid;
background-color: rgb(22, 15, 121);
padding:2px;
border:1px solid;
border-color: white;
border-radius:12px;
	font: 14pt &quot;Ericsson Hilda&quot;;
}

#template_checks_btn:hover{
background-color:rgb(255,255,255);
border:1px solid;
border-color:rgb(22, 15,121 );
color:rgb(22, 15, 121);
	font: 700 14pt &quot;Ericsson Hilda&quot;;
}

#template_checks_btn:pressed{
background-color:rgb(220, 220, 220);
border:1px solid;
border-color:rgb(15, 13, 125);
color:rgb(22, 15, 121);
	font: 800 14pt &quot;Ericsson Hilda&quot;;
}</string>
                  </property>
                  <property name="text">
                   <string>Template Checks</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="running_config_pre_checks_btn">
                  <property name="minimumSize">
                   <size>
                    <width>260</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>260</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Ericsson Hilda</family>
                    <pointsize>14</pointsize>
                    <italic>false</italic>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">#running_config_pre_checks_btn{
color:white;
background:solid;
background-color: rgb(22, 15, 121);
padding:2px;
border:1px solid;
border-color: white;
border-radius:12px;
	font: 14pt &quot;Ericsson Hilda&quot;;
}

#running_config_pre_checks_btn:hover{
background-color:rgb(255,255,255);
border:1px solid;
border-color:rgb(22, 15,121 );
color:rgb(22, 15, 121);
	font: 700 14pt &quot;Ericsson Hilda&quot;;
}

#running_config_pre_checks_btn:pressed{
background-color:rgb(220, 220, 220);
border:1px solid;
border-color:rgb(15, 13, 125);
color:rgb(22, 15, 121);
	font: 800 14pt &quot;Ericsson Hilda&quot;;
}</string>
                  </property>
                  <property name="text">
                   <string>Running Config Pre Checks</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="cli_preparation_btn">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">#cli_preparation_btn{
color:white;
background:solid;
background-color: rgb(22, 15, 121);
padding:2px;
border:1px solid;
border-color: white;
border-radius:12px;
	font: 14pt &quot;Ericsson Hilda&quot;;
}

#cli_preparation_btn:hover{
background-color:rgb(255,255,255);
border:1px solid;
border-color:rgb(22, 15,121 );
color:rgb(22, 15, 121);
	font: 700 14pt &quot;Ericsson Hilda&quot;;
}

#cli_preparation_btn:pressed{
background-color:rgb(220, 220, 220);
border:1px solid;
border-color:rgb(15, 13, 125);
color:rgb(22, 15, 121);
	font: 800 14pt &quot;Ericsson Hilda&quot;;
}</string>
                  </property>
                  <property name="text">
                   <string>CLI Preparation</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="running_config_post_checks_btn">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>260</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>260</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">#running_config_post_checks_btn{
color:white;
background:solid;
background-color: rgb(22, 15, 121);
padding:2px;
border:1px solid;
border-color: white;
border-radius:12px;
	font: 14pt &quot;Ericsson Hilda&quot;;
}

#running_config_post_checks_btn:hover{
background-color:rgb(255,255,255);
border:1px solid;
border-color:rgb(22, 15,121 );
color:rgb(22, 15, 121);
	font: 700 14pt &quot;Ericsson Hilda&quot;;
}

#running_config_post_checks_btn:pressed{
background-color:rgb(220, 220, 220);
border:1px solid;
border-color:rgb(15, 13, 125);
color:rgb(22, 15, 121);
	font: 800 14pt &quot;Ericsson Hilda&quot;;
}</string>
                  </property>
                  <property name="text">
                   <string>Running Config Post Checks</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="status_labels_horizontal_layout">
                <item>
                 <widget class="QLabel" name="template_checks_label">
                  <property name="font">
                   <font>
                    <family>Ericsson Hilda</family>
                    <pointsize>14</pointsize>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Hello</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="running_config_pre_checks_label">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Ericsson Hilda</family>
                    <pointsize>14</pointsize>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Hello</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="cli_preparation_status_label">
                  <property name="font">
                   <font>
                    <family>Ericsson Hilda</family>
                    <pointsize>14</pointsize>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Hello</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="running_config_post_checks_label">
                  <property name="font">
                   <font>
                    <family>Ericsson Hilda</family>
                    <pointsize>14</pointsize>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Hello</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Application_GUI.qrc"/>
 </resources>
 <connections/>
</ui>
